tests/test_defects.py

```diff
@@ -701,8 +698,7 @@ def test_badImage(self):
         config.nPixBorderLeftRight = 0
 
         task = cpPipe.defects.MeasureDefectsTask(config=config)
-        defects = task.findHotAndColdPixels(testImage, [config.nSigmaBright,
-                                                        config.nSigmaDark])
+        defects = task.findHotAndColdPixels(testImage)
```

Comment:
I don't see any tests for the combined tasks, or the `VALUE` option.

Reply:
I added a new file for the combined defects task that repeats the tests in the existing defects task and added a couple of tests to the `VALUE` option.